src/variants.ini

```diff
@@ -339,6 +339,12 @@ capturesToHand = true
 dropChecks = false
 whiteDropRegion = *1 *2 *3 *4 *5
 blackDropRegion = *6 *7 *8 *9 *10
+mobilityRegionWhiteFers = d1 f1 e2 d3 f3
+mobilityRegionBlackFers = d8 f8 e9 d10 f10
+mobilityRegionWhiteElephant = c1 g1 a3 e3 i3 c5 g5
+mobilityRegionBlackElephant = c6 g6 a8 e8 i8 c10 g10
+mobilityRegionWhiteSoldier = a4 a5 c4 c5 e4 e5 g4 g5 *6 *7 *8 *9 *10
```

Comment:
Upon a closer look it seems like the i file squares are missing here.

Reply:
It should be fixed now. Thank you! Hopefully, this is the last error.